net/DevExtreme.AspNet.Data.Tests/RemoteGroupingTests.cs

```diff
@@ -233,6 +233,29 @@ public class RemoteGroupingTests {
             Assert.Equal(1, groups.Count());
             Assert.Equal(2, result.groupCount);
         }
+
+        [Fact]
+        public void GroupDateTime_NullDates() {
```

Comment:
It's strange that two "sibling" `GroupDateTime_NullDates` test cases use different data and different number of groups.